Lib/multiprocessing/managers.py

```diff
@@ -312,7 +315,7 @@ def fallback_repr(self, conn, ident, obj):
     def dummy(self, c):
         pass
 
-    def debug_info(self, c):
+    def debug_info(self, c): # Perhaps include debug info about 'c'?
```

Comment:
Just a style thing, but I would not put such a suggestion beside the function declaration, rather below the docstring.

Reply:
No problem.